src/validations/guild.validation.ts

```diff
@@ -17,13 +22,14 @@ const updateGuild = {
             channelId: Joi.string().required(),
             channelName: Joi.string().required()
         })),
-        period: Joi.date()
+        period: Joi.string()
```

Comment:
it's timestamp, right ?

Reply:
yes. I change it for a test and forgot to back it to date.